ui/raidboss/data/06-ew/trial/zodiark-ex.ts

```diff
@@ -1,23 +1,297 @@
 import NetRegexes from '../../../../../resources/netregexes';
+import Outputs from '../../../../../resources/outputs';
+import { callOverlayHandler } from '../../../../../resources/overlay_plugin_api';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  activeSigils: { x: number; y: number; typeId: string; npcId: string }[];
+  activeFrontSigils: { x: number; y: number; typeId: string; npcId: string }[];
+  paradeigmaCounter: number;
+  adikiaCounter: number;
+}
+
+const directionOutputStrings = {
+  northeast: Outputs.northeast,
```

Comment:
consider also `Outputs.dirNE` for the shorter `NE` vs `Northeast`?

Reply:
I don't like the short "NE" style callouts but if there is multiple like in Algedon they might indeed be better because of their briefness.

I've changed it to dirNE style.

Reply:
And dirN for north too!